### ISSUE TITLE
sort dependencies?

### ISSUE THREAD

**Issue by hannesm** [ocaml/opam2web#173]
when I browse packages that require a huge amount of dependencies (such as http://opam.ocaml.org/packages/vpnkit/), it'd be much more readable if the dependencies would be lexicographically sorted (maybe respecting the `build` / `doc` / `test` tags?). WDYT?

**Comment by avsm** [comment]
yes agreed; and this would help spot duplicates.  It might be better off done in the opam pretty printer though, and a mechanical regeneration of the repository,

**Comment by rjbou** [comment]
Sorting all fields is a good idea. I tend to not generalize it to all outputs (integrating it in the pretty-printer directly), but having a function in `OpamFileTools` than sorts all concerned fields. Like that, it can be called by external programs (opam2web, ocaml.org, camelus, etc.). Also adding an option to `opam show --sort` that outputs the sorted opam file.
